.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+updates:
+    # Reference to the package manager using the relevant YAML value
+  - package-ecosystem: "github-actions"
+    # Location of the package manifests
+    directory: "/"
+    schedule:
+      # How frequently the Dependabot will check for updates
+      interval: "weekly"
+      # Day on which updates are checked for
+      day: "monday"
+    commit-message:
+      # Prefix attached to commit messages with relevant reference to package
+      prefix: "chore(actions): "
+    # List of users permitted to review and approve Dependabot PRs
+    reviewers:
+      - "davidmogar"
+      - "johnbieren"
+      - "theflockers"
+      - "scoheb"
+      - "happybhati"
+      - "mmalina"
+      - "kasemAlem"
+      - "Troy876"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+    commit-message:
+      prefix: "chore(go): "
+    reviewers:
+      - "davidmogar"
+      - "johnbieren"
+      - "theflockers"
+      - "scoheb"
+      - "happybhati"
+      - "mmalina"
+      - "kasemAlem"
+      - "Troy876"
+      
```

Comment:
It still seems that there's something going on in the last line. Could you fix it?  The empty line is not needed, just the character.

Reply:
I've removed the last line and I believe I've rebased main, everything should be fine now.

Reply:
Fixed the empty line issue.